package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "react-calculator",
+  "version": "1.0.0",
+  "description": "In the initial commit to this project you have been provided with only an ESLint config for a React project. Your task is to create an interactive React Calculator page that responds to user input.",
+  "main": ".eslintrc.js",
+  "dependencies": {
+    "react-dom": "^16.13.1",
+    "react": "^16.13.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.9.6",
+    "@babel/preset-react": "^7.9.4",
+    "babel-loader": "^8.1.0",
+    "eslint": "^7.0.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.20.0",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.11"
+  },
+  "scripts": {
+    "dev": "./node_modules/.bin/webpack --progress --inline --config webpack.dev.js",
+    "lint": "./node_modules/.bin/eslint --format codeframe --ext jsx .",
```

Comment:
This is a useful command, you should try running it

![image](https://user-images.githubusercontent.com/5775716/83044874-c9940d00-a012-11ea-95b9-d5b700053064.png)

if you had you could have fixed these issues

![image](https://user-images.githubusercontent.com/5775716/83045105-25f72c80-a013-11ea-8a5a-5ae5082d87af.png)